src/main/java/training/subsystems/roller/Roller.java

```diff
@@ -0,0 +1,54 @@
+package training.subsystems.roller;
+
+import com.revrobotics.CANSparkBase;
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import utils.GBSubsystem;
+
+public class Roller extends GBSubsystem {
+
+	private static CANSparkMax motor;
```

Comment:
final and not static
⡿⢋⣮⢻⣿⣿⣿⣿⣿⣿⣿⠿⠿⠛⠛⠛⠿⢿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿ 
⣇⢸⣿⣀⢿⣿⣿⣿⠿⢋⡀⠀⠀⠀⠀⠀⠀⠀⠈⠙⢿⣿⣿⣿⣿⣿⣿⣿⣿⣿ 
⣿⣿⣿⣿⣮⡻⠟⣁⣾⠿⠻⢷⣦⠀⠀⠀⠀⠀⠀⠀⠀⠹⣿⣿⣿⣿⣿⣿⣿⣿ 
⣿⣿⣿⣿⡿⢿⣾⣿⣿⣷⣦⣤⣽⣷⠀⠀⠀⠀⠀⣀⢀⡀⠹⣿⣿⣿⣿⣿⣿⣿ 
⣿⣿⣿⡿⢁⠾⠿⣿⣋⣽⣦⠨⢿⣿⡄⠀⠀⣒⡛⠋⠙⠛⠆⣿⣿⣿⡿⡋⠉⠹ 
⣿⣿⡿⠁⠀⠀⠀⠈⠑⠂⠁⠀⠚⠋⠀⠀⢀⡽⣫⡥⢀⡀⠀⠙⠋⠁⣾⣇⠃⢀ 
⣿⡿⢡⣾⣦⣄⣀⣠⣶⣿⣤⡆⠀⠀⠀⠀⠀⠳⡛⠃⠀⠁⠀⢠⣶⣿⣿⣿⣿⣿ 
⣿⠁⠈⠉⣸⣿⣿⠛⠻⠿⢿⣷⣄⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢸⣿⣿⣿⣿⣿⣿ 
⣿⣶⣷⣶⣿⣿⡙⢷⡄⠀⠀⠀⠉⠉⠉⠉⠛⠂⢆⠀⠀⠀⠀⢸⣿⣿⣿⣿⣿⣿ 
⣿⣿⣿⣿⣿⣿⣿⣶⣍⠂⢄⡀⠀⠀⠀⠀⠀⠀⠈⠣⠀⠀⠀⢸⣿⣿⣿⣿⣿⣿ 
⣿⣿⣿⣿⣿⣿⣿⣿⡇⠀⠀⠈⠉⠑⠒⠂⠀⠀⠀⠀⠀⠀⠀⣿⣿⣿⣿⣿⣿⣿ 
⣿⣿⣿⣿⣿⣿⠿⠿⠷⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣸⣿⣿⣿⣿⣿⣿⣿ 
⣿⣿⣿⣿⣿⣿⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣼⣿⣿⣿⣿⣿⣿⣿⣿ 
⣿⣿⣿⣿⣿⣿⣿⣆⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣠⣾⣿⣿⣿⣿⣿⣿⣿⣿⣿ 
⣿⣿⣿⣿⣿⣿⣿⣿⣿⣶⣤⣄⣀⣀⣀⣀⣤⣶